moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryInterimID.js

```diff
@@ -0,0 +1,143 @@
+import React, { useEffect, useState } from "react";
+import { Box, Grid, Icon, TextField, Typography } from "@material-ui/core";
+
+import ProjectSummaryLabel from "./ProjectSummaryLabel";
+
+import {
+  PROJECT_UPDATE_INTERIM_ID,
+  PROJECT_CLEAR_INTERIM_ID,
+} from "../../../../queries/project";
+import { useMutation } from "@apollo/client";
+
+/**
+ * ProjectSummaryInterimID Component
+ * @param {Number} projectId - The id of the current project being viewed
+ * @param {Object} data - The data object from the GraphQL query
+ * @param {function} refetch - The refetch function from apollo
+ * @param {Object} classes - The shared style settings
+ * @param {function} snackbarHandle - The function to show the snackbar
+ * @returns {JSX.Element}
+ * @constructor
+ */
+const ProjectSummaryInterimID = ({
+  projectId,
+  data,
+  refetch,
+  classes,
+  snackbarHandle,
+}) => {
+  // Instantiate Original Value
+  const [originalValue, setOriginalValue] = useState(
+    data?.moped_project?.[0]?.interim_project_id ?? null
```

Comment:
This is tricky because, if you stay in the summary view, this seems to work. If you navigate away and return, the value clears because the `interim_project_id` is missing from `SUMMARY_QUERY`. 

It's a straightforward fix but I noticed that the value still doesn't populate. ~It might have to do with the duplicate state above? Not sure.~ I _think_ the next comment is where this is happening.

Reply:
you're right, it was the next comment where it was happening, you cant find the length of a number i.e. `44.length`, thats undefined. so it was always displaying None